### PR TITLE
ovito: 3.7.11 -> 3.11.0

### PR DESCRIPTION
**Pull request by CHN-beta**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/graphics/ovito/default.nix [hunk 1]
```diff
@@ -48,16 +53,43 @@ mkDerivation rec {
...
+        terminal = false;
+        startupNotify = false;
+        desktopName = "ovito";
+        startupWMClass = "Ovito";
+        categories = [ "Science" ];
+      };
+    in ''
+      mkdir -p $out/share/applications
+      substituteAll ${desktopFile}/share/applications/${pname}.desktop $out/share/applications/${pname}.desktop
+    '';
```
Comment (Aleksanaa) [inline pkgs/applications/graphics/ovito/default.nix:82]:
```suggestion
  # manually create a desktop file
  desktopItems = [ (makeDesktopItem {
        name = "ovito";
        comment= "Open Visualization Tool";
        exec = "ovito";
        icon = "ovito";
        terminal = false;
        startupNotify = false;
        desktopName = "ovito";
        startupWMClass = "Ovito";
        categories = [ "Science" ];
      })];
  
  postInstall = let
      icon = fetchurl {
        url = "https://www.ovito.org/wp-content/uploads/logo_rgb-768x737.png";
        hash = "sha256-FOmIUeXem+4MjavQNag0UIlcR2wa2emJjivwxoJh6fI=";
      };
  in ''
    install -Dm644 ${icon} $out/share/pixmaps/ovito.png
  '';
```

File: pkgs/applications/graphics/ovito/default.nix [hunk 2]
```diff
@@ -17,21 +18,25 @@
...
     owner = "stuko";
     repo = "ovito";
     rev = "v${version}";
-    hash = "sha256-Z3uwjOYJ7di/LLllbzdKjzUE7m119i03bA8dJPqhxWA=";
+    hash = "sha256-egiA6z1e8ZS7i4CIVjsCKJP1wQSRpmSKitoVTszu0Mc=";
   };
 
   nativeBuildInputs = [
     cmake
+    wrapQtAppsHook
```
Comment (Aleksanaa) [inline pkgs/applications/graphics/ovito/default.nix:39]:
```suggestion
    wrapQtAppsHook
    copyDesktopItems
```

File: pkgs/applications/graphics/ovito/default.nix [hunk 3]
```diff
@@ -48,16 +55,42 @@ mkDerivation rec {
...
     qtsvg
     qttools
+    qtwayland
   ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
     VideoDecodeAcceleration
   ];
 
+  # manually create a desktop file
+  desktopItems = [ (makeDesktopItem {
+        name = "ovito";
```
Comment (Aleksanaa) [inline pkgs/applications/graphics/ovito/default.nix:65]: weird indentation

Comment (CHN-beta) [reply pkgs/applications/graphics/ovito/default.nix]: Done, big thanks.

**Comment by twhitehead** [comment]
Thanks everyone! :+1:

**Pull request merged**